commands/parse/parse_test.go

```diff
@@ -81,9 +82,11 @@ func TestInputFlagForparseInput(t *testing.T) {
 		viper.Reset()
 		viper.Set("input", testunit.input)
 		parsed, _ := parseInput(ctx, testunit.fileList)
+		tmp := string(parsed)
+		tmp = strings.Replace(tmp, "\\r", "", -1)
```

Comment:
Maybe we should just add this to `parseInput` ?

Reply:
I got the perspective you mentioned. I'm going to add this to the method itself.